### PR TITLE
Run pull request summarizer on Travis CI

### PR DESCRIPTION
**Pull request by chrismytton**
This adds an extra rake task to the Travis CI build. When the task is run it will add a comment to the current pull request containing a summary of changes.

Similar to #16718 
Related to https://github.com/everypolitician/everypolitician/issues/449

### CODE REVIEW

File: Rakefile.rb [hunk 1]
```diff
@@ -67,3 +67,13 @@ def json_write(file, json)
     puts "Pull request #{pull_request.number} is outdated. (Newest pull request is #{pull_request.superseded_by.number})"
   end
 end
+
+desc "[Travis only] Post a summary of the pull request that's currently being build"
```
Comment (tmtmtmtm) [inline Rakefile.rb]: typo "build" → "built"

Comment (chrismytton) [inline Rakefile.rb]: Gah, every time! This typo will be with me for life 😢

File: Rakefile.rb [hunk 2]
```diff
@@ -67,3 +67,24 @@ def json_write(file, json)
     puts "Pull request #{pull_request.number} is outdated. (Newest pull request is #{pull_request.superseded_by.number})"
   end
 end
+
+desc "Post a summary of the pull request that's currently being built"
+task :pull_request_summary do
+  if ENV['TRAVIS_PULL_REQUEST'] == 'false'
```
Comment (tmtmtmtm) [inline Rakefile.rb]: I think this would all be a lot cleaner if we had two separate tasks: one for travis to run, and one for humans to run.

Comment (chrismytton) [inline Rakefile.rb]: Yep, good point. I'll merge this now so I can shut off the Heroku bits, then split that up as a separate pull request.

**Comment by tmtmtmtm** [comment]
dancing_penguins

**Pull request merged**